client/src/App.js

```diff
@@ -18,25 +18,113 @@ import './App.css';
 import Landing from './views/Landing';
 import Home from './views/Home';
 import AccountPage from './views/Account';
+import Review from './views/Review';
 
 //@ utils
 import * as ROUTES from './Routes/routes';
 
+//@theme
+import Theme from './components/Styling/Theme';
 
-const theme = {
+const hivestack =     
```

Comment:
Is this the same theme that's in index2.js and theme.js? At first glance they look identical. If so I'm sure the one from theme.js could be imported 👍

Reply:
Yes, I tried this. It's not working at the moment, which is why I have them in all files that needs it until I can fix it.